tests/components/mold_indicator/test_config_flow.py

```diff
@@ -0,0 +1,117 @@
+"""Test the Mold indicator config flow."""
+
+from __future__ import annotations
+
+from unittest.mock import AsyncMock
+
+from homeassistant import config_entries
+from homeassistant.components.mold_indicator.const import (
+    CONF_CALIBRATION_FACTOR,
+    CONF_INDOOR_HUMIDITY,
+    CONF_INDOOR_TEMP,
+    CONF_OUTDOOR_TEMP,
+    DEFAULT_NAME,
+    DOMAIN,
+)
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+
+async def test_form_sensor(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+    """Test we get the form for sensor."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["step_id"] == "user"
+    assert result["type"] is FlowResultType.FORM
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_NAME: DEFAULT_NAME,
+            CONF_INDOOR_TEMP: "sensor.indoor_temp",
+            CONF_INDOOR_HUMIDITY: "sensor.indoor_humidity",
+            CONF_OUTDOOR_TEMP: "sensor.outdoor_temp",
+            CONF_CALIBRATION_FACTOR: 2.0,
+        },
+    )
+    await hass.async_block_till_done()
```

Comment:
```suggestion
```

Reply:
As we're asserting the mocking call below we should keep this to not make it flaky

Reply:
Nope, the call directly returns a config entry, so there is nothing to wait for here

Reply:
I mean `mock_setup_entry` (which mocks the `setup_entry` function in `__init__`) which I presume is loaded after a config entry has been made? (see line 55)

Reply:
Oh like that, I've never thought about that, I've been removing these for a while now and I never have seen any problems with flakiness tbh

Reply:
Maybe it's done as part of saving the config entry and therefore it's already awaited. I'll just remove it as I couldn't get it to fail locally either.